src/redux/gas.ts

```diff
@@ -350,15 +362,16 @@ export const gasPricesStartPolling = (network = Network.mainnet) => async (
   const getOptimismGasPrices = async () => {
     const provider = await getProviderForNetwork(Network.optimism);
     const baseGasPrice = await provider.getGasPrice();
-    const gasPriceGwei = Number(weiToGwei(baseGasPrice.toString()));
+    const normalGasPrice = weiToGwei(baseGasPrice.toString());
 
     const priceData = {
-      average: gasPriceGwei,
-      avgWait: 0.5,
-      fast: gasPriceGwei,
-      fastWait: 0.2,
-      safeLow: gasPriceGwei,
-      safeLowWait: 1,
+      fast: normalGasPrice,
+      fastWait: 0.34,
+      normal: normalGasPrice,
+      // 20 secs
+      normalWait: 0.34,
+      urgent: normalGasPrice,
+      urgentWait: 34,
```

Comment:
is this supposed to be 0.34 like the other `normalWait` and `fastWait` ?

Reply:
oh yeah, we're not using those values but I'll change it 👌